src/ASM/ASMs2D.C

```diff
@@ -3361,7 +3361,7 @@ void ASMs2D::BasisFunctionCache::internalCleanup ()
 
 bool ASMs2D::BasisFunctionCache::setupQuadrature ()
 {
-  int p[3];
+  int p[3]{};
   patch.getOrder(p[0],p[1],p[2]);
```

Comment:
Same here, but now for the `getOrder()` method which as the same weakness as `getSize()`